ui/webui/src/components/app.jsx

```diff
@@ -71,18 +72,18 @@ export const Application = () => {
             startEventMonitorStorage({ dispatch });
 
             startEventMonitorLocalization({ dispatch });
+
+            getIsFinal().then(
```

Comment:
Is setting Beta moved to the bus.address change handler intentionally?

Reply:
I noticed other DBus calls there, so I put it there. Placing it where the "old" bootstamp checking code was did not work at all - I guess that place might be run somehow before the DBus machinery is initialized ?

Reply:
Indeed - the place that this was before  is not guaranteed to run after address is set.